grammars/less.cson

```diff
@@ -354,6 +336,17 @@
   }
 ]
 'repository':
+  'booleans':
+    # true is truthy, everything else is false: there is no 'false' keyword
+    # http://lesscss.org/features/#mixin-guards-feature-guard-comparison-operators
+    'match': '\\btrue\\b'
+    'name': 'constant.language.boolean.less'
+  'variables':
+    'match': '(@|\\-\\-)[\\w-]+(?=\\s*)'
```

Comment:
This lookahead seems rather useless since it can potentially match nothing.

Reply:
Good catch. Not sure what I was thinking there...